commit 816aa478d79cfaddd085609c2340e92f28f90ded
Author: Rob Johnston <rob.johnston@joyent.com>
Date:   2019-06-05T22:49:18+00:00 (4 months ago)
    
    OS-7821 mandoc should allow ERRORS section in all man section 7 variants

diff --git a/usr/src/cmd/mandoc/mdoc_validate.c b/usr/src/cmd/mandoc/mdoc_validate.c
index 52545e3c46..32067baf0a 100644
--- a/usr/src/cmd/mandoc/mdoc_validate.c
+++ b/usr/src/cmd/mandoc/mdoc_validate.c
@@ -3,7 +3,7 @@
  * Copyright (c) 2008-2012 Kristaps Dzonsons <kristaps@bsd.lv>
  * Copyright (c) 2010-2018 Ingo Schwarze <schwarze@openbsd.org>
  * Copyright (c) 2010 Joerg Sonnenberger <joerg@netbsd.org>
- * Copyright 2018, Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
  *
  * Permission to use, copy, modify, and distribute this software for any
  * purpose with or without fee is hereby granted, provided that the above
@@ -2428,9 +2428,9 @@ post_sh_head(POST_ARGS)
 	case SEC_ERRORS:
 		if (*mdoc->meta.msec == '4')
 			break;
-		if (*mdoc->meta.msec == '7' && *(mdoc->meta.msec + 1) == 'I')
+		if (*mdoc->meta.msec == '7')
 			break;
-		goodsec = "2, 3, 4, 7I, 9";
+		goodsec = "2, 3, 4, 7, 9";
 		/* FALLTHROUGH */
 	case SEC_RETURN_VALUES:
 	case SEC_LIBRARY:
